Raven.StorageExporter/ConsoleUtils.cs

```diff
@@ -18,14 +18,17 @@ public static void ConsoleWriteLineWithColor(ConsoleColor color, string message,
 
         public static void PrintErrorAndFail(string ErrorMessage, string stackTrace = null)
         {
-            Console.Clear();
-            ConsoleUtils.ConsoleWriteLineWithColor(ConsoleColor.Red, ErrorMessage);
-            if (stackTrace != null)
+            if (Console.IsOutputRedirected == false)
             {
-                ConsoleUtils.ConsoleWriteLineWithColor(ConsoleColor.Blue, "StackTrace:\n" + stackTrace);
+                Console.Clear();
+                ConsoleWriteLineWithColor(ConsoleColor.Red, ErrorMessage);
+                if (stackTrace != null)
+                {
+                    ConsoleWriteLineWithColor(ConsoleColor.Blue, "StackTrace:\n" + stackTrace);
```

Comment:
Should't we use Environment.NewLine here?

Reply:
Also, shouldn't we _only_ avoid calling `Console.Clear()` since that was the problem? The above change also won't print the error message to the console so it won't show in the details of 4.0 studio. The user won't be able to figure out why the process has failed.

Reply:
Both addressed in https://github.com/ravendb/ravendb/pull/4718